posts/Fabio/2018-02-13-hibernate-ogm-5-3-CR1-released.adoc

```diff
@@ -0,0 +1,58 @@
+= "Hibernate OGM 5.3 CR1 is out"
+Fabio Massimo Ercoli
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We're happy to announce new release:
+http://hibernate.org/ogm/releases/5.3/#get-it[Hibernate OGM 5.3 CR1]
+
+All the changes are described in the
+https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10160&version=31626
```

Comment:
You're missing the title of the link. I suppose it should be release notes?

Reply:
Yes. I will add the title. Thank you